panacea/tx.go

```diff
@@ -12,12 +12,12 @@ import (
 )
 
 type TxBuilder struct {
-	client     GrpcClientI
+	client     QueryClient
 	marshaller *codec.ProtoCodec
 }
 
-func NewTxBuilder(client GrpcClientI) *TxBuilder {
-	marshaller := codec.NewProtoCodec(client.GetInterfaceRegistry())
+func NewTxBuilder(client QueryClient) *TxBuilder {
+	marshaller := client.cdc
 
 	return &TxBuilder{
```

Comment:
In my opinion, it would be clearer to use `client.cdc` directly without declaring the `marshaller` variable.
```go
return &TxBuilder {
  client: client,
  marshaller: client.cdc,
}

```

Reply:
Agree.

Reply:
Thanks, I fixed it in [f7b488e](https://github.com/medibloc/panacea-doracle/pull/46/commits/f7b488e1dd94f4dfc16eac4e9b5036b9ba09d41c).